sammba/data_fetchers/func.py

```diff
@@ -162,22 +162,24 @@ def fetch_zurich_test_retest(subjects=range(15), sessions=[1], data_dir=None,
         for a in anat:
             corrected_a = os.path.join(os.path.dirname(a),
                                        '3DRARE_corrected.nii.gz')
-            _reset_affines(a, corrected_a,
-                           axes_to_permute=[(1, 2)],
-                           axes_to_flip=[0],
-                           verbose=0)
-            corrected_anat.append(corrected_a)
+            if not os.path.exists(corrected_a):                           
+                _reset_affines(a, corrected_a,
+                               axes_to_permute=[(1, 2)],
+                               axes_to_flip=[0],
+                               verbose=0)
+                corrected_anat.append(corrected_a)
```

Comment:
this should go out of the if section, because we want to get the path in any case